crates/goose-server/src/logging.rs

```diff
@@ -66,6 +66,7 @@ pub fn setup_logging(name: Option<&str>) -> Result<()> {
 
     // Create console logging layer for development - INFO and above only
     let console_layer = fmt::layer()
+        .with_writer(std::io::stderr)
```

Comment:
This change goes a bit further than strictly necessary as it moves all console logging to stderr, while we only truly needed to do that for mcp servers. But IMO we should always prefer stderr to stdout for logging.